release-notes/opendistro-for-elasticsearch-security.release-notes-1.13.1.1.md

```diff
@@ -28,7 +28,7 @@ Compatible with Elasticsearch 7.10.2
 
 ### Maintenance
 
-* Upgrade CXF and jackson-binding ([#1943](https://github.com/opensearch-project/security/pull/1943))
+* Upgrade CXF ([#1943](https://github.com/opensearch-project/security/pull/1943))
```

Comment:
It would be inconsistent with the PR title: https://github.com/opensearch-project/security/pull/1943. Would you please update the PR title too?

Reply:
Updated